homeassistant/components/adguard/__init__.py

```diff
@@ -127,9 +123,7 @@ async def async_unload_entry(
 ) -> bool:
     """Unload AdGuard Home config entry."""
     unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
-    if unload_ok:
-        hass.data[DOMAIN] -= 1
-    if hass.data[DOMAIN] == 0:
+    if not hass.config_entries.async_entries(DOMAIN):
```

Comment:
Here's an example of how to check for loaded config entries:
https://github.com/home-assistant/core/blob/94a66fa64896a19ca657b8da9cc238ddcbb6cda6/homeassistant/components/simplisafe/__init__.py#L426-L435